contracts/weapons.sol

```diff
@@ -642,6 +653,12 @@ contract Weapons is Initializable, ERC721Upgradeable, AccessControlUpgradeable {
         }
     }
 
+    function setBaseWeaponMultiplier(int128 multiplier) public restricted {
+        require(multiplier >= 0, "BaseWeaponMultiplier too low");
```

Comment:
These require statements will always block the function.
int128 variables aren't inherently handled as fixed64x64 numbers so they can't be used for a fractional comparison
I also don't think these sanity checks should be kept at all.
I'd rather the function parameter took in a decimal number that we then divide by like 10k using ABDK to get the fraction we need.